src/client/package.json

```diff
@@ -15,7 +15,6 @@
     "@types/react": "^16.9.35",
```

Comment:
Link "Learn more about the azure free account..." doesn´t seem to work. Note to myself: Try again in the installed extension

Reply:
jaja ni caso ! tenía un breakpoint !